service/src/main/resources/application.yml

```diff
@@ -101,6 +101,14 @@ workspace:
     - # A special spend profile with no billing account associated with it.
       id: no-billing-account
 
+  application:
+    configurations:
+      identifier: 6397C5FF-F83C-4837-95DB-059198FD89BF
+      name: Leonardo
+      description: Leonardo application SA
+      service-account: leonardo-dev@broad-dsde-dev.iam.gserviceaccount.com
```

Comment:
Typically, we make the application.yml the production state and override for other environments. It helps prevents accidental exposure of test stuff.

Note that you can inject this into the configuration via `scripts/write-config.sh`. It makes different settings/gets different secrets for different environments. It generates a file that is optionally included application.yml.